.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: ci testing
+
+
+on:
+  push:
+    branches: master
+  pull_request:
+
+
+jobs:
+
+  run_test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [
+          3.7,
+          3.8,
+          3.9,
+        ]
+        include:
+          - python-version: 3.7
+            tox-env: py37
+          - python-version: 3.8
+            tox-env: py38
+          - python-version: 3.9
+            tox-env: py39
+
+    steps:
+
+      - name: cancel previous workflows that are still running
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: checkout latest commit
+        uses: actions/checkout@v2
+
+      - name: setup python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+
+      - name: linting and code style
+        if: matrix.python-version == '3.7'
+        run: tox -vv -e lint -- --color=yes
+
+      - name: tests and coverage
+        run: tox -vv -e ${{ matrix.tox-env }} -- --color=yes
```

Comment:
Is `matrix.tox-env` able to find the `tox-env` values nested inside `matrix.include.[*].tox-env`?

Reply:
Yes! I also tried with `${{ matrix.tox-env}}` but it failed—here it should be w/o brackets.

Reply:
> I also tried with ${{ matrix.tox-env }} but it failed

Is there a typo? Because this is what is in the file.